src/api/xxAMIDOxx.xxSTACKSxx.API/xxAMIDOxx.xxSTACKSxx.API.csproj

```diff
@@ -20,15 +20,16 @@
     <PackageReference Include="Amido.Stacks.API" Version="0.1.85" />
     <PackageReference Include="Amido.Stacks.API.Swagger" Version="0.1.113" />
     <PackageReference Include="Amido.Stacks.Application.CQRS.Abstractions" Version="0.1.83" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.7" />
       
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.12.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.12.0" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc4" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="5.0.0-rc4" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="5.0.0-rc4" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="5.0.0-rc4" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="5.0.0-rc4" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.5.1" />
```

Comment:
the package `Amido.Stacks.API.Swagger` also uses the version `Version="5.0.0-rc4"`, if you could update it as well will be nice

Reply:
Created separate PR for this in stacks-packages repo: https://github.com/amido/stacks-packages/pull/24/files